cypress/e2e/homepage.cy.js

```diff
@@ -13,6 +13,12 @@ describe('Homepage', () => {
     cy.get('.region').should('have.length', 9)
     cy.get('.region').first().contains('Alaska')
     cy.get('.region').last().contains('Mid-Atlantic')
+    cy.get('a').contains('Park Tracker').click()
+    cy.url().should('eq', 'http://localhost:3001/parks/tracker')  
```

Comment:
This just reminded me that I need to check my URLs in my test! Great work on this testing 😄